src/test/regress/upgrade/config.py

```diff
@@ -1,68 +1,277 @@
 from os.path import expanduser
+import random
+import socket
+from contextlib import closing
+import os
+import upgrade_common as common
 
+COORDINATOR_NAME = "coordinator"
+WORKER1 = "worker1"
+WORKER2 = "worker2"
 
-BEFORE_PG_UPGRADE_SCHEDULE = './before_pg_upgrade_schedule'
-AFTER_PG_UPGRADE_SCHEDULE = './after_pg_upgrade_schedule'
+REGULAR_USER_NAME = "regularuser"
+SUPER_USER_NAME = "postgres"
 
-AFTER_CITUS_UPGRADE_COORD_SCHEDULE = './after_citus_upgrade_coord_schedule'
-BEFORE_CITUS_UPGRADE_COORD_SCHEDULE = './before_citus_upgrade_coord_schedule'
-MIXED_BEFORE_CITUS_UPGRADE_SCHEDULE = './mixed_before_citus_upgrade_schedule'
-MIXED_AFTER_CITUS_UPGRADE_SCHEDULE = './mixed_after_citus_upgrade_schedule'
+CUSTOM_TEST_NAMES = ["custom_sql_test", "custom_create_test"]
```

Comment:
Manually adding test names to `CUSTOM_TEST_NAMES ` is hard. We already write the names to the schedules, can't we read from there?

Reply:
Yeah we should be able to do that and that is a nice point as that would be quite helpful when adding tests, I will do that.